Objects/Main.java

```diff
@@ -0,0 +1,15 @@
+package Objects;
+import Objects.*;
+
+public class Main {
```

Comment:
Why is there a Main.java
I don't understand the purpose of this. is it necessary?